career-growth.md

```diff
@@ -0,0 +1,69 @@
+# Conscious Career Growth
+
+It used to be the case that people would stay with one employer for their entire careers. This is no longer the case today. It is normal and expected for most people to move from role to role throughout their lives. That said, you are at Packback right now for a reason! Your job with Packback is an alliance that benefits both parties:
+
+* You apply your unique skills, vision and expertise to move Packback forward as a company.
+* Your work at Packback pushes you to learn new skills, develop your vision and build new expertise.
```

Comment:
This is also really pedantic, but can we use the Oxford comma and add `,` after each item before an `and` ("vision")? We use it in the sentence below, so IMO we should be consistent with its usage.

Reply:
I studied journalism in college, and AP style doesn't use Oxford commas. I always forget it.